src/extension/app/store/app.js

```diff
@@ -234,18 +234,19 @@ export class AppStore {
       return;
     }
 
-    const { profile } = this.status;
-    const { authorized } = this.siteStore;
+    const { status: configStatus } = this.siteStore;
     const code = this.status?.code?.status === 200;
     const media = this.status?.webPath?.match(/\/media[_-]+\d+/)
       && this.status?.preview?.status === 404
       && this.status?.live?.status === 404
       && this.status?.code?.status === 404;
 
-    if (!profile && !authorized) {
+    if (configStatus === 401) {
       this.state = STATE.LOGIN_REQUIRED;
-    } else if (!authorized) {
+    } else if (configStatus === 403) {
       this.state = STATE.UNAUTHORIZED;
+    } else if (configStatus >= 404) {
+      this.state = STATE.ERROR;
```

Comment:
I don't see any use of `STATE.ERROR` in `plugin-action-bar.js`. I think we will need that or else it loads an empty sidekick when a 404 is returned from config.

![Screenshot 2025-06-23 at 9 28 38 AM](https://github.com/user-attachments/assets/176010af-d8d3-463b-bb69-f1ea98252a2b)